src/value_converter.rs

```diff
@@ -818,3 +850,72 @@ pub fn build_python_from_serde_value(
     }
 }
+
+/// Convert python List, Tuple or Set into geo `Point`.
+///
+/// # Errors
+/// May return error if cannot convert Python type into Rust one.
+#[allow(clippy::needless_pass_by_value)]
+pub fn build_point(value: Py<PyAny>) -> RustPSQLDriverPyResult<Point> {
+    Python::with_gil(|gil| {
+        let bind_value = value.bind(gil);
+        if bind_value.is_instance_of::<PyList>()
+            | bind_value.is_instance_of::<PyTuple>()
+            | bind_value.is_instance_of::<PySet>()
+        {
+            let mut result_vec: Vec<f64> = vec![];
+            let params = bind_value.extract::<Vec<Py<PyAny>>>()?;
+
+            if params.len() != 2 {
+                return Err(RustPSQLDriverError::PyToRustValueConversionError(
+                    "PyPoint supports only pair of int/float combination.".to_string(),
+                ));
+            }
+
+            for inner in params {
+                let inner_bind = inner.bind(gil);
+
+                if inner_bind.is_instance_of::<PyFloat>() | inner_bind.is_instance_of::<PyInt>() {
+                    let python_dto = py_to_rust(inner_bind)?;
+                    match python_dto {
+                        PythonDTO::PyIntI16(pyint) => result_vec.push(pyint.try_into().unwrap()),
+                        PythonDTO::PyIntI32(pyint) => result_vec.push(pyint.try_into().unwrap()),
+                        PythonDTO::PyIntI64(_) => {
+                            return Err(RustPSQLDriverError::PyToRustValueConversionError(
+                                "Not implemented this type yet".into(),
+                            ))
+                        }
+                        PythonDTO::PyIntU32(pyint) => result_vec.push(pyint.try_into().unwrap()),
+                        PythonDTO::PyIntU64(_) => {
+                            return Err(RustPSQLDriverError::PyToRustValueConversionError(
+                                "Not implemented this type yet".into(),
+                            ))
+                        }
+                        PythonDTO::PyFloat32(pyfloat) => {
+                            result_vec.push(pyfloat.try_into().unwrap())
+                        }
+                        PythonDTO::PyFloat64(pyfloat) => result_vec.push(pyfloat),
+                        _ => {
+                            return Err(RustPSQLDriverError::PyToRustValueConversionError(
+                                "Incorrect types of point coordinates. It must be int or float"
+                                    .into(),
+                            ))
+                        }
+                    };
+                } else {
+                    return Err(RustPSQLDriverError::PyToRustValueConversionError(
+                        "PyPoint supports only list/tuple/set pair of int/float combination."
+                            .to_string(),
+                    ));
+                }
+            }
+
+            Ok(point!(x: result_vec[0], y: result_vec[1]))
+        } else {
```

Comment:
No need for the `else` word, you can just write at the end of the function.
```
return Err(RustPSQLDriverError::PyToRustValueConversionError(
                "PyPoint must have pair int/float values combination passed in tuple, list or set."
                    .to_string(),
            ));
```

Reply:
You can use reverse clause as I mentioned before in different functions.

Reply:
Removed `else` keywoard. And for reverse clause i think that in this section it will be worse, because `if condition` consist of multiple functions that return bool values joined with `or`, so that will be a little more uncomfortable to read.